gdx/src/com/badlogic/gdx/Audio.java

```diff
@@ -100,4 +100,7 @@ public interface Audio {
 	 *
 	 * @return A array of available output devices */
 	public String[] getAvailableOutputDevices ();
+
```

Comment:
Just out of curiosity, would there be any advantage/disadvantage of extending Audio with Disposable?
I.e. "Audio extends Disposable"?

Reply:
For a small interface and for just one method I personally prefer to see all the methods of the interface in one place when I inspect it, but it's a personal taste. I'll extend Disposable that seems to be the preferred option.